packages/ethereum-wallets/src/lib/index.ts

```
@@ -528,6 +541,47 @@ const EthereumWallets: WalletBehaviourFactory<
     transactions: Array<Optional<Transaction, "signerId" | "receiverId">>
   ) => {
     const nearTxs = await transformTransactions(transactions);
+    if (bannedNearAddressesPackage === null) {
+      await importBannedNearAddressesPackage();
+    }
+
+    let restrictedActionError: string | null = null;
+    for (let i = 0; i < nearTxs.length; i++) {
+      for (let y = 0; y < nearTxs[i].actions.length; y++) {
+        const action = nearTxs[i].actions[y];
+        if (action.type !== "FunctionCall") continue
```

ESLint

[error] Expected { after 'if' condition.

(curly)